documentation/guides/breakpoints-migration.md

```diff
@@ -0,0 +1,792 @@
+## New breakpoints
+
+TODO: write about the 5 new breakpoints
+
+### Media queries variables
+
+We created some Sass variables to handle the new breakpoints and most of the migration you need to do is replacing old media-queries or mixins with them in your code.
```

Comment:
```suggestion
We've created a collection of Sass variables for applying responsive styles at each breakpoint alias. A majority of the migration will involve replacing legacy media-queries or mixins with the following values:
```

Reply:
Aaron and I workshopped this slightly to follow some of the language conventions we've been using throughout the project and in previous migration guides.

Reply:
Yeah sure! Feel free to make suggestions of any kind!